config/jobs/kubernetes-sigs/cluster-api/cluster-api-periodics-main.yaml

```diff
@@ -2,8 +2,6 @@ periodics:
 - name: periodic-cluster-api-test-main
   interval: 1h
   decorate: true
```

Comment:
overlooked again :(
should have added the fields from https://github.com/kubernetes/test-infra/pull/27156, and remove the service account preset from the jobs in #27156 as well
@killianmuldoon